src/components/Dashboard/StaffNotation.vue

```diff
@@ -71,6 +87,17 @@
                 <span class="app-blue">Record Conversion</span>
               </v-list-item-title>
             </v-list-item>
+            <v-list-item v-if="isFirm && isHistorical" @click="showPutBackOnDialog()" :disabled="!isHistorical">
```

Comment:
Should this also be disabled when prop `disabled` is True?

Reply:
Ok I get it now.

I recommend using a v-template for `isFirm`, and in that block put these 2 v-ifs (isHistorial and !isHistorical). I think that might more clearly bundle this functionality and make it easier to understand. What do you think?

Reply:
yes that makes sense, the change has been made